
builtin_interfaces/Time stamp
float32 angular_rate      # angular rate in rad/s

# TOPICS offboard/controller_input_angular_rate 
